src/main/scala/org/deepdive/datastore/MysqlDataStore.scala

```diff
@@ -56,7 +56,7 @@ class MysqlDataStore extends JdbcDataStore with Logging {
       case "text" | "varchar" => s"convert(${expr.toString()}, char)"
       // in mysql, convert to unsigned guarantees bigint.
       // @see http://stackoverflow.com/questions/4660383/how-do-i-cast-a-type-to-a-bigint-in-mysql
-      case "bigint" | "int" => s"convert(${expr.toString()}, unsigned)"
+      case "bigint" | "int" => s"convert(${expr.toString()}, signed)"
```

Comment:
Casting to signed in mysql should be also bigint. 
A related post is http://dba.stackexchange.com/questions/57363/why-does-cast1-as-signed-integer-return-a-bigint-on-mysql.

I'm installing mysql to confirm this.